packages/caver-transaction/src/transactionTypes/abstractTransaction.js

```diff
@@ -65,7 +77,6 @@ class AbstractTransaction {
 
         // The variables below are values that the user does not need to pass to the parameter.
         if (createTxObj.nonce !== undefined) this.nonce = createTxObj.nonce
-        if (createTxObj.gasPrice !== undefined) this.gasPrice = createTxObj.gasPrice
```

Comment:
Do we need to `fillTransaciton()` function in each transaction? How about checking whether createTxObj is DynamicFeeTx or not?